releasenotes/notes/user-defined-filename-for-account-710cb32e812b3bd8.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    Allow the user to store account details in a file specified by the user. The default remains 
+    `~/.qiskit/qiskit-ibm.json`.
+
```

Comment:
Can you add an example here of how to use a different filename? I think it would be helpful